src/cook-web/src/app/c/[[...id]]/page.module.css

```
@@ -0,0 +1 @@
+.newChatPage{width:100%;height:100vh;height:100dvh;display:flex;flex-flow:row nowrap;font-size:var(--font-size-4);font-weight:var(--font-weight-4-1);color:var(--font-color-main)}.newChatPage.pc{display:block}.newChatPage.pad{display:block}.newChatPage.m{display:block}.newChatPage .chatMobileHeader{position:fixed;top:0;left:0;z-index:2;width:100vw}
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

height is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)